client/src/client/user_interface/__init__.py

```diff
@@ -160,6 +161,12 @@ def __init__(
 
         self.loop.create_task(self.view.render_all())
 
+        self.public_key: Optional[str] = None
```

Comment:
Could we refer to it as user id? Or would it cause too much confusion at UI level? Since this is not technically the public key, but only its hash